site/content/entry/yuzu-progress-report-aug-2022/index.md

```diff
@@ -0,0 +1,334 @@
+For example:
+
+```
+Before: here             to here
+         v                 v
+12345678 12345678 // comment
+         ^      ^
+After:  here  to here
+```
+
+Our current file system emulation is spaghetti code caused by old guesses made when less information was available, and patchwork done by developers that are no longer contributing to the project. 
+There’s no way to sugar-coat it, this is the main reason Morph has been working so hard for over a year on `Project Gaia`, a full rewrite of the current virtual file system implementation.
+
+Meanwhile, some patchwork can be done to ease the head pain on developers and modders alike.
+For example, `Build IDs` are padded by a bunch of zeroes, and modders are more used to the short form without including the irrelevant zeroes at the beginning.
+To alleviate this, byte[] {{< gh-hovercard "8790" "parses the Build ID string using fmt" >}} to dynamically add the required zeroes when needed.
+
+Continuing to focus on improving RetroArch’s experience in yuzu, vonchenplus discovered that recursively calculating the size of the files when parsing the directories has a negative effect on performance, in this case perceived as “loading times” when booting a game with the emulator (inside an emulator).
+{{< gh-hovercard "8804" "Simply ignoring this information" >}} is a good way to improve the user experience.
+Note that this change doesn’t help in running native Switch games on yuzu, it only provides faster load times for RetroArch.
+
+## UI changes
+
+People have been asking for stronger audio volume in yuzu for a long time, and Maide delivers.
+Now, if the user goes to `Emulation > Configure… > Audio`, they can {{< gh-hovercard "8756" "set the volume up to 200%," >}} really cranking it to 11!
+
+{{< imgs
+	"./audio.png| Don't wake up your neighbours"
+  >}}
+
+Something to keep in mind is that some games have their volume tuned up to the output they are using.
+A game may be louder in handheld over docked, for example, so don’t stress your poor ears.
+
+Part of yuzu’s code is still based on [Citra](https://citra-emu.org/), even some sections that are no longer relevant, or have become outdated over the years.
+Cleaning up code is a boring but necessary part of any project, more so for open source ones, and yuzu is no exception.
+One feature that was very useful in the early days is loading ELF binaries.
+Instead of dealing with XCI and NSP dumps, back in the early days, devs dumped and extracted the whole file structure of the game, and loaded the ELF binary directly, in a similar way to how on Windows the user opens a .exe file to load a program.
+
+Some years down the line and not only this feature [tarnished out of grace](https://www.youtube.com/watch?v=6R3ouGNcACQ), but it also stopped working.
+Since it’s not a priority at the moment, and it can lead to user confusion, byte[] {{< gh-hovercard "8701" "removed it from the File submenu." >}}
```

Comment:
```suggestion
Some years down the line and not only is this feature [tarnished out of grace](https://www.youtube.com/watch?v=6R3ouGNcACQ), but it also stopped working.
Since it’s not a priority at the moment, and it can lead to user confusion, byte[] {{< gh-hovercard "8701" "removed it from the File dropdown." >}}
```